docs-website/sidebars.js

```diff
@@ -448,12 +448,12 @@ module.exports = {
       // "docs/wip/tags-guide", -- not needed
       // "docs/wip/personal-access-tokens-guide", -- not needed
 
-      {
-        "Advanced Tutorials": [
-          // "docs/wip/advanced-guides",
-          // "docs/wip/search-ranking",
-        ],
-      },
+      // {
+      //   "Advanced Tutorials": [
```

Comment:
@maggiehays fyi - these sections weren't showing up in the sidebar before either, but now docusaurus complains about them so I commented them out